### ISSUE TITLE
Users unable to log in

### ISSUE THREAD

**Issue by EliWei** [layersoflondon/application#661]
Some users are unable to log into their accounts, while others have no issue. Logging in from the admin interface is not a problem, nor is creating and then logging into a new account.

**Comment by EliWei** [comment]
Doesn't work trying to re-set password

**Comment by EliWei** [comment]
I've also tried logging in on different browsers but all end up with the error 524 - timeout.

=> I think 504 errors imply data problems. Maybe there’s some kind of horrible recursive query we are doing, which only affects people who own a lot of records?

**Comment by EliWei** [comment]
Update from Adam on the 12 Aug:
Following this morning's meeting, as you're planning to work on this in the coming weeks, another email's come in regarding the issue of logging in and I've been running a few more tests:

This account (-username removed-) only has 5 records but he's unable to sign in.
I signed in as him from the admin and then tried to sign out, but that led to this:
![image (2)](https://user-images.githubusercontent.com/12016064/90134490-89bb0700-dd71-11ea-8f88-5e4526f3a3e5.png)

**Comment by EliWei** [comment]
Doing the same process with other accounts with only a few records works fine.
I can sign in and log out using the IHR staff account (layersoflondon@london.ac.uk)

With my personal gmail account, once logged in via the admin pages, I then changed my password to see if that would change things. Unfortunately, now it doesn't recognise it when I try to sign in, so I'm not getting the timed out screen but I'm getting nothing at all - just the login in screen as if my password was wrong.
When I went to the "forgot password" to get an email for password reset, that took ages and then produced the error 524 page:
![image (3)](https://user-images.githubusercontent.com/12016064/90134602-b5d68800-dd71-11ea-99fd-01279c8cd3da.png)
